bpo-36651: Fixed Asyncio Event Loop documentation inconsistency

naccht opened the pull request:
<!--
Thanks for your contribution!
Please read this comment in its entirety. It's quite important.

# Pull Request title

It should be in the following format:

```
bpo-NNNN: Summary of the changes made
```

Where: bpo-NNNN refers to the issue number in the https://bugs.python.org.

Most PRs will require an issue number. Trivial changes, like fixing a typo, do not need an issue.

# Backport Pull Request title

If this is a backport PR (PR made against branches other than `master`),
please ensure that the PR title is in the following format:

```
[X.Y] <title from the original PR> (GH-NNNN)
```

Where: [X.Y] is the branch name, e.g. [3.6].

GH-NNNN refers to the PR number from `master`.

-->

# [bpo-36651](https://bugs.python.org/issue36651): Fixed Asyncio Event Loop documentation inconsistency  

In the documentation for the call_later and the call_at methods there is a note which says that the delay cannot be longer than a day, but both methods have a note saying that this limitation was removed in Python 3.8
Here I fixed this issue by removing the pre-exising note and added a versionchanged. 

To test my changes I have rebuilt the documentation with ```make html```. I did not have any errors and the effected page displayed correctly on a browser.


<!-- issue-number: [bpo-36651](https://bugs.python.org/issue36651) -->
https://bugs.python.org/issue36651
<!-- /issue-number -->

naccht committed:
bpo-36651: Fixed Asyncio Event Loop documentation inconsistency

blurb-it[bot] committed:
📜🤖 Added by blurb_it.

blurb-it[bot] committed:
📜🤖 Added by blurb_it.

tirkarthi commented on Misc/NEWS.d/next/Documentation/2019-04-17-20-21-32.bpo-36651.3iUjvY.rst:
```diff
@@ -0,0 +1 @@
+bpo-36651: Fixed Asyncio Event Loop documentation inconsistency
```
There is an error in Travis due to this blurb entry at https://travis-ci.org/python/cpython/jobs/521431229#L309 . I am not sure this doc change needs a news entry.

```
blurb.BlurbError: Error in Misc/NEWS.d/next/Documentation/2019-04-17-20-21-32.bpo-36651.3iUjvY.rst:0:
Blurb 'body' can't start with 'bpo-'!
```

naccht committed:
Delete 2019-04-17-20-21-32.bpo-36651.3iUjvY.rst

Removing file not passing bot checking

asvetlov committed:
Drop changenote

The change is small and self-descriptive. There is no need for changelog record in this case
